getting_started/install_openshift.adoc

```diff
@@ -199,7 +199,7 @@ basic authentication, user access, and routes.
 {product-title} provides two command line utilities to interact with it.
 
 * `oc`: for normal project and application management
-* `oc adm` or `oadm`: for administrative tasks
+* `oadm` or `oc adm`: for administrative tasks
```

Comment:
`oadm` was deprecated in 3.9. You should remove it from 3.9 on.

Reply:
If I'm reading `diff` between branches correctly, this looks like it is correct in 3.11, but not in master. I'll dig around some more and resolve cherry-pick conflicts as needed.